bullet_train-api/app/helpers/api/open_api_helper.rb

```diff
@@ -71,7 +71,7 @@ def automatic_components_for(model, locals: {})
       }.merge(locals))
 
       schema_json = jbuilder.json(
-        model.new,
+        FactoryBot.example(model.name.underscore.to_sym) || model.new,
```

Comment:
@newstler I'm not very familiar with what BT does with OpenAPI, so please excuse me if this is a dumb question. But I'm wondering if the method is ever used in production or production-like environments. The reason I ask is becasue `factory_bot_rails` is current in the `:development, :test` group in the `Gemfile` and would not be available in production, which would then cause this method to fail.

Reply:
After looking at this more closely I see that we use `FactoryBot` all over this file, so I assume that my previously-stated concern is invalid.